src/calculateRentalCost.js

```diff
@@ -4,7 +4,26 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentDay = 40;
+  let fullAmount = 0;
```

Comment:
Use 'const' instead of 'let' for 'fullAmount' since its value is not being reassigned after the initial calculation. This adheres to the checklist item 3, preferring 'const' over 'let' where possible.